cmd/start.go

```diff
@@ -144,5 +149,53 @@ func newStartCommand(ctx context.Context, args []string) *cobra.Command {
 }
 
 
+func encryptConfig(encryptPass string) error{
+	var s = make([]string,2)
+	s[0] = "password:"
+	s[1] = "encrypted:" + encryptPass
+	confPath := filepath.Join(viper.GetString(flagFileServerPath),defaultConfigFileName)
+	confLines, err := readLines(confPath)
+	if err != nil {
+		return err
+	}
+	for i:=0; i<len(confLines);i++{
+		if strings.Contains(confLines[i], "password:"){
+			confLines[i] = strings.Join(s," ")
+		}
+	}
+	err = writeLines(confLines, confPath)
+	if err != nil {
+		return err
+	}
+	return nil
+}
 
+func readLines(path string) ([]string, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
 
+	var lines []string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+	return lines, scanner.Err()
+}
+
+// writeLines writes the lines to the given file.
+func writeLines(lines []string, path string) error {
+	file, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
```

Comment:
catch the error even when the call is deferred by surrounding the call to Close in a anonymous function:

defer func() {
 if err := file.Close(); err != nil {
  logger.withError(err).Error("error closing config file after writing");
 }
}()

Reply:
Done